pkg/apis/openebs.io/v1alpha1/types.go

```diff
@@ -13,14 +13,17 @@ import (
 // StoragePoolClaim describes a StoragePoolClaim.
 type StoragePoolClaim struct {
 	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
+	metav1.ObjectMeta `json:"metadata",omitempty`
```

Comment:
what was the issue with the above line?

Reply:
That was just for learning-trial and error. will revert